projects/js-packages/publicize-components/src/hooks/use-share-limits/index.ts

```diff
@@ -1,123 +1,103 @@
 import { useSelect } from '@wordpress/data';
-import { __, sprintf } from '@wordpress/i18n';
+import { __, _n, sprintf } from '@wordpress/i18n';
 import { store as socialStore } from '../../social-store';
 
 export type ShareLimits = {
 	status: 'approaching' | 'exceeded' | 'full' | 'none';
-	noticeType: 'default' | 'highlight' | 'warning' | 'error';
+	noticeType: 'default' | 'warning' | 'error';
 	message: string;
+	usedCount: number;
+	scheduledCount: number;
+	remainingCount: number;
+};
+
+export type UseShareLimitsArgs = {
+	scheduledCountAdjustment?: number;
+	usedCountAdjustment?: number;
 };
 
 /**
  * Returns the messages for the share limits
  *
- * @param {number} shareLimit - Share limit
- *
+ * @param {number} remainingCount - The number of shares left
  * @returns {ReturnType<typeof getMessages>} Share limits messages
  */
-export function getMessages( shareLimit: number ) {
+export function getMessages( remainingCount: number ) {
+	const remaining = Number.isFinite( remainingCount )
+		? sprintf(
+				// translators: %d: The number of shares to social media remaining
+				_n(
+					'You have %d auto-share remaining.',
+					'You have %d auto-shares remaining.',
+					remainingCount,
+					'jetpack'
+				),
+				remainingCount
+		  )
+		: '';
 	return {
-		default: sprintf(
-			// translators: %1$d is the number of shares allowed in 30 days.
-			__( 'Share limit for 30 days: %1$d.', 'jetpack' ),
-			shareLimit
-		),
+		default: remaining,
 		exceeded: __(
-			'You have exceeded your share limit. Your posts will no longer be shared.',
+			'You have have reached your auto-share. Scheduled posts will not be shared until shares become available.',
```

Comment:
This looks like a typo:
```suggestion
			'You have reached your auto-share limit. Scheduled posts will not be shared until shares become available.',
```

Reply:
Fixed. Thanks